docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md

```diff
@@ -0,0 +1,199 @@
+
+1. Create endpoints for two public IPs in Azure and Azure Stack. Select `External endpoints` as the endpoint type. For the location of Azure Stack endpoint, select the nearest location.
+
+    ```
+    az network traffic-manager endpoint create --name AzureEndpoint --profile-name CloudFoundryTrafficManagerProfile --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --type externalEndpoints --endpoint-location WestUS2 --target <PUBLIC_IP_ADDRESS_OF_LOAD_BALANCER_IN_AZURE>
+    az network traffic-manager endpoint create --name AzureStackEndpoint --profile-name CloudFoundryTrafficManagerProfile --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --type externalEndpoints --endpoint-location WestUS --target <PUBLIC_IP_ADDRESS_OF_LOAD_BALANCER_IN_AZURE_STACK>
+    ```
+
+See more details in the [doc](https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-create-profile).
+
+## Create an Azure DNS Zone
+
+1. Create a DNS zone
+
+    ```
+    export ZONE_NAME="mslovelinux.com"
+    az network dns zone create --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --name ${ZONE_NAME}
+    ```
+
+1. Create DNS records
+
+    1. Create records for `azure.mslovelinux.com` and `azurestack.mslovelinux.com`.
+
+        ```
+        az network dns record-set a create --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --name "*.azure"
+        az network dns record-set a add-record --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --record-set-name "*.azure" --ipv4-address <PUBLIC_IP_ADDRESS_OF_LOAD_BALANCER_IN_AZURE>
+        az network dns record-set a create --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --name "*.azurestack"
+        az network dns record-set a add-record --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --record-set-name "*.azurestack" --ipv4-address <PUBLIC_IP_ADDRESS_OF_LOAD_BALANCER_IN_AZURE_STACK>
+        ```
+
+    1. Create records for `*.mslovelinux.com`.
+
+        ```
+        az network dns record-set cname create --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --name "*"
+        az network dns record-set cname set-record --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --record-set-name "*" --cname "${TRAFFIC_MANAGER_DNS_NAME}.trafficmanager.net"
+        ```
+
+1. Confiure DNS server to Azure DNS
+
+    1. Get your Name Server Domain Name. You can get four `nsdname` from the output.
+
+        ```
+        az network dns record-set ns show --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --name "@"
+        ```
+
+    1. Configure DNS server. It may differ from each provider, but the process is basically same. You can usually get the guidance in your domain-name service provider about how to configure your name server domain name.
```

Comment:
2. ?

Reply:
and also for other places.

Reply:
Numbered lists do not need to sequentially increase. For example, 
The following content
```
1. a
1. b
1. c
```
will be rendered to:
1. a
1. b
1. c

Reply:
cool. learned that.